### PR TITLE
CASMTRIAGE-6677: add flow clarification

### PR DESCRIPTION
**Pull request by bo-quan**
# Description
Added CSM service upgrade flow clarification depending on different scenarios. When IUF is used to upgrade CSM with additional products, don't proceed to Stage 2; only proceed to Stage 2 for CSM only upgrade without IUF.

# Checklist

- [ ] If I added any command snippets, the steps they belong to follow the prompt conventions (see [example][1]).
- [ ] If I added a new directory, I also updated `.github/CODEOWNERS` with the corresponding team in [Cray-HPE][2].
- [x] My commits or Pull-Request Title contain my JIRA information, or I do not have a JIRA.

 
[1]: https://github.com/Cray-HPE/docs-csm/blob/main/introduction/documentation_conventions.md#using-prompts
[2]: https://github.com/Cray-HPE/teams

### CODE REVIEW

File: upgrade/Stage_1.md [hunk 1]
```diff
@@ -83,4 +83,7 @@ For any typescripts that were started during this stage, stop them with the `exi
 
 ## Stage completed
 
-This stage is completed. Continue to [Stage 2](Stage_2.md).
+This stage is completed. There are two different upgrade paths moving forward:
+
+Path 1: For CSM only upgrade without IUF, continue to [Stage 2](Stage_2.md).
+Path 2: For [upgrade CSM and additional products with IUF](../operations/iuf/workflows/upgrade_csm_and_additional_products_with_iuf.md), press the Back button in your web browser and perform the next step.
```
Comment (rustydb) [inline upgrade/Stage_1.md]:
```suggestion
- Path 1: For CSM only upgrade without IUF, continue to [Stage 2](Stage_2.md).
- Path 2: For [upgrade CSM and additional products with IUF](../operations/iuf/workflows/upgrade_csm_and_additional_products_with_iuf.md), press the Back button in your web browser and perform the next step.
```

File: upgrade/Stage_1.md [hunk 2]
```diff
@@ -83,4 +83,7 @@ For any typescripts that were started during this stage, stop them with the `exi
 
 ## Stage completed
 
-This stage is completed. Continue to [Stage 2](Stage_2.md).
+This stage is completed. There are two different upgrade paths moving forward:
+
+- Path 1: For CSM only upgrade without IUF, continue to [Stage 2](Stage_2.md).
+- Path 2: For [upgrade CSM and additional products with IUF](../operations/iuf/workflows/upgrade_csm_and_additional_products_with_iuf.md), press the Back button in your web browser and perform the next step.
```
Comment (nrockershousen) [inline upgrade/Stage_1.md]: What is the intended path after going back in the web browser? Can we be more specific about what stage to continue with.

Comment (bo-quan) [reply upgrade/Stage_1.md]: As the steps may change in the iuf workflow, I don't want to specify the next step here but I have made a change by following the language used in https://cray-hpe.github.io/docs-csm/en-15/operations/iuf/workflows/image_preparation/

**Review by nrockershousen — APPROVED** [review]

**Review by rustydb — APPROVED** [review]

**Pull request merged**